WebView/app/build.gradle

```diff
@@ -63,6 +63,15 @@ dependencies {
     implementation "com.google.android.material:material:$material_components_version"
 
     testImplementation 'junit:junit:4.13'
+    testImplementation 'androidx.test:core:1.0.0'
+    testImplementation 'org.mockito:mockito-core:1.10.19'
+
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
-}
+    androidTestImplementation 'androidx.test.espresso:espresso-web:3.1.0'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-contrib:3.0.2'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-intents:3.0.2'
+    androidTestImplementation 'androidx.test:rules:1.1.1'
+    androidTestImplementation 'androidx.test:runner:1.1.0'
+    androidTestImplementation "androidx.webkit:webkit:$webkit_version"
```

Comment:
nit: I don't think we need this because we're already including this with `implementation` above.

Reply:
Ok